benches/benches.rs

```diff
@@ -14,24 +10,70 @@ use cosmian_findex::InMemory;
 
 #[cfg(feature = "sqlite-mem")]
 use cosmian_findex::SqliteMemory;
+#[cfg(feature = "sqlite-mem")]
+const SQLITE_PATH: &str = "./target/benches.sqlite";
 
 #[cfg(feature = "redis-mem")]
 use cosmian_findex::RedisMemory;
+/// Gets the Redis URL to use based on the following priority:
+/// 1. If REDIS_HOST environment variable is set, use "redis://{REDIS_HOST}:6379"
+/// 2. If no environment variable is set but a Redis server is reachable at
+///    localhost (127.0.0.1:6379), use "redis://127.0.0.1:6379"
+/// 3. If neither of the above conditions are met, fall back to the default
+///    dockered Redis service at "redis://redis:6379"
+#[cfg(feature = "redis-mem")]
+fn get_redis_url() -> String {
```

Comment:
I prefer the old version since we do not hard-code anything but a default. The CI is to be fixed accordingly. Except if the old version cannot work, I would rather like you to revert this change.

Reply:
My point was that there should be no need to hard-code `redis://redis:6379`. Plus your new function has too much intelligence : since it attempts connecting to `localhost` first. Previous version was simpler: return the URL in the environment variable, defaulting to `localhost`. But it was not less generic since any address could passed through this environment variable.

In short, imho, you made a function more complex in order to fix something that should not be fixed here.